src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -64,9 +64,13 @@ protected ReadThreadingAssemblerArgumentCollection getReadThreadingAssemblerArgu
 
     @Override
     public ReadThreadingAssembler createReadThreadingAssembler(){
-        if(mitochondria && assemblerArgs.pruningLog10OddsThreshold == ReadThreadingAssemblerArgumentCollection.DEFAULT_PRUNING_LOG_ODDS_THRESHOLD) {
-            assemblerArgs.pruningLog10OddsThreshold = DEFAULT_MITO_PRUNING_LOG_ODDS_THRESHOLD;
+        if(mitochondria ) {
+            assemblerArgs.recoverAllDanglingBranches = true;
```

Comment:
Is there a way to turn this off in mitochondria mode? Or is this such an obvious solution for mitochondria (and obviously not the right solution for somatic) that it's not exposed to the user as an option?

Reply:
I expect it to be obvious for mitochondria because with high depth 1) there are always going to be forks within your dangling tails and 2) adaptive pruning works better and therefore we don't expect M2 to go haywire chasing down incorrect dangling ends.

As long as your validations look good I am in favor of not exposing.

For somatic it's probably the right solution, too, but it unearths mapping errors that we have never had to deal with before.  Regenerating and M2 pon with this option turned on, among other solutions, would probably catch it.